packages/paste-core/primitives/combobox/package.json

```diff
@@ -34,6 +34,7 @@
     "@twilio-paste/dropdown-library": "^1.1.1",
     "prop-types": "^15.7.2",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "@testing-library/user-event": "^13.2.1"
```

Comment:
Technically this is a root dependency as jest runs from the root of the repo, and not at the package level.

See readme https://github.com/twilio-labs/paste#dev-deps. It's certainly not consistent across the repo

Reply:
Ah okey cool, I will confirm that we have it there, if not I'll add. Either way, will remove from the package. Thanks!